_tools/release_docs/main.go

```diff
@@ -36,8 +36,9 @@ import (
 var (
 	branch       = flag.String("branch", "", "The git branch from which to pull docs. (e.g. release-1.0, master).")
 	outputDir    = flag.String("output-dir", "", "The directory in which to save results.")
-	remote       = flag.String("remote", "upstream", "The name of the remote repo from which to pull docs.")
-	apiReference = flag.Bool("apiReference", true, "Whether update api reference")
+	version      = flag.String("version", "", "The release version. It should be the same as the version segment in the URL, e.g., when importing docs that will be hosted at k8s.io/v1.0/, the version flag should be \"v1.0\".")
```

Comment:
Sorry I just noticed, we should have updated the "usage:" line at the top (line 18)